packages/analyticsdx-template-lint/src/schemas/readiness-schema.json

```diff
@@ -312,6 +366,14 @@
             "filters": []
           }
         },
+        {
+          "label": "New DataCloudRowCount definition",
+          "body": {
+            "type": "DataCloudRowCount",
+            "object": "${0}",
+            "filters": []
+          }
+        },
```

Comment:
Sorry similar question, is the reason why we don't provide a default snippet for accessCheck also that it's an internal feature? :))

Reply:
Yep.